torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -612,7 +632,7 @@ def _partition_parameters(
                     params_sorted = sorted(param_group["params"], key=lambda t: t.numel(), reverse=True)
                     for param in params_sorted:
                         # Greedily add the parameter to rank with smallest size so far
-                        rank = sizes.index(min(sizes))
+                        rank = self._get_min_index(sizes)
```

Comment:
Since I was using `_get_min_index()` for the new overlapping approach, I thought I might as well change this here as well.